components/testing/runners/BUILD

```diff
@@ -1,5 +1,12 @@
 load("@build_bazel_rules_apple//apple/testing/default_runner:ios_test_runner.bzl", "ios_test_runner")
 
+ios_test_runner(
+    name = "IPHONE_4S_IN_9_0",
+    device_type = "iPhone 4s",
+    os_version = "9.0",
```

Comment:
Tip: This requires the team to install a 9.0 simulator in addition to 9.3.  Do you think it's worth adding that requirement?

Reply:
Ah right, this needs to be conditional on the environment.

Reply:
I think as part of this change, it's worth updating our documentation to direct contributors to download Xcode 10.2, install the 9.3 and 9.0 simulators, and then restore whatever Xcode version they are using.  What do you think?

Reply:
I don't feel that's a reasonable expectation for our contributors our ourselves; we should be able to rely on the CI to test + run this simulator.

Reply:
I understand your point, and I also like having CI being robust.  However, it's hard for me to work on debugging a broken test (or update one) when I cannot reproduce the test environment. My request is only to add instructions to explain to contributors how to reproduce the environment locally, which we already do for installing Bazel.

Let's discuss this offline and see if we can agree on a path forward.

Reply:
+1 happy to clarify our docs in that regard and agreed it's out-of-scope for this change.

Reply:
Spoke offline and discussed adding the docs as part of this change. Moving PR to WIP until that lands.